crates/core/src/file_group/reader.rs

```diff
@@ -75,8 +75,8 @@ impl FileGroupReader {
     /// Creates a new reader with the given base URI and options.
     ///
     /// # Arguments
-    ///     * `base_uri` - The base URI of the file group's residing table.
-    ///     * `options` - Additional options for the reader.
+    /// * `base_uri` - The base URI of the file group's residing table.
+    /// * `options` - Additional options for the reader.
```

Comment:
Please ignore these changes while reviewing. When I ran `cargo test --workspace` there were quite a few doctests that failed locally. So, I had to fix them.